### PR TITLE
feat: add gemini code sample and test for text-only input

### PR DESCRIPTION
**Pull request by irataxy**
## Description

Fixes b:336785030

- [X] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: vertexai/snippets/src/main/java/vertexai/gemini/TextInput.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            + " which would go down in history as the world's most famous mouse. Yes, Mickey makes"
+            + " his debut here, in this cheery tale of life on board a steamboat. The animation is"
+            + " good for it's time, and the plot - though a little simple - is quite jolly. A true"
+            + " classic, and if you ever manage to get it on video, you won't regret it.";
+
+    String output = textInput(projectId, location, modelName, textPrompt);
+    System.out.println(output);
+  }
+
+  // Analyzes the provided text input.
```
Comment (Sita04) [inline vertexai/snippets/src/main/java/vertexai/gemini/TextInput.java:54]: Would it okay to make this description a little more elaborate?
Unsure of what "analyzes" refers to here.

**Review by Sita04 — APPROVED** [review]
Thanks, LGTM.
Will wait for Eric's approval to merge.

File: vertexai/snippets/src/main/java/vertexai/gemini/TextInput.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * laishers. "A New Era in Cartoon History." Review of Steamboat Willie.
+ * 4 January 2001. https://www.imdb.com/review/rw0005574/?ref_=rw_urv
+ */
```
[the remark this answers is not in the document]
Comment (irataxy) [reply vertexai/snippets/src/main/java/vertexai/gemini/TextInput.java:20]: Yes, this is a citation for the movie review.

File: vertexai/snippets/src/main/java/vertexai/gemini/TextInput.java (hunk 1 above)
Comment (irataxy) [reply vertexai/snippets/src/main/java/vertexai/gemini/TextInput.java]: Done

File: vertexai/snippets/src/test/java/vertexai/gemini/SnippetsIT.java [hunk 3]
```diff
@@ -180,26 +182,42 @@ public void testQuickstart() throws IOException {
...
+            + " (1 is most negative, 10 is most positive, 5 will be neutral). Include an"
+            + " explanation.\n"
+            + "This era was not just the dawn of sound in cartoons, but of a cartoon character"
+            + " which would go down in history as the world's most famous mouse. Yes, Mickey makes"
+            + " his debut here, in this cheery tale of life on board a steamboat. The animation is"
+            + " good for it's time, and the plot - though a little simple - is quite jolly. A true"
+            + " classic, and if you ever manage to get it on video, you won't regret it.";
+    String output = TextInput.textInput(PROJECT_ID, LOCATION, GEMINI_PRO, textPrompt);
+    assertThat(output).isNotEmpty();
+    assertThat(output).contains("Explanation");
```
[the remark this answers is not in the document]
Comment (irataxy) [reply vertexai/snippets/src/test/java/vertexai/gemini/SnippetsIT.java]: Thanks! Should be fixed now.

**Review by gericdong — APPROVED** [review]

**Pull request merged**